### PR TITLE
Decompilation of code_805D8C8.s Part 1

### PR DESCRIPTION
**Pull request by DizzyEggg**
Part1: Decomp of `code_805D8C8.s` 

Failed function if someone wants to take a look: https://decomp.me/scratch/96Sci

### CODE REVIEW

File: src/code_805D8C8_1.c [hunk 1]
```diff
@@ -0,0 +1,4835 @@
...
+        }
+        else {
+            PrintFormatStringOnWindow(0x10, y, gUnknown_8106BF4, windowId, 0);
+        }
+    }
+
+    sub_80073E0(windowId);
+}
+
+//
```
Comment (AnonymousRandomPerson) [inline src/code_805D8C8_1.c:4835]: Empty comment.

File: src/code_805D8C8_1.c [hunk 2]
```diff
@@ -0,0 +1,4835 @@
...
+"pop {r4-r7}\n"
+"pop {r0}\n"
+"bx r0\n"
+".align 2, 0\n"
+"_0805E2C0: .4byte gDungeon\n");
+}
+
+#endif
+
+/* Nope
```
Comment (AnonymousRandomPerson) [inline src/code_805D8C8_1.c:1879]: Is this the start of the function? If we want to merge this in then let's change this comment to be more descriptive.

File: src/code_805D8C8_1.c [hunk 3]
```diff
@@ -0,0 +1,4835 @@
...
+extern const u8 *gUnknown_80FDE18;
+extern const u8 *gUnknown_80F8B24;
+extern const u8 *gTeamToolboxAPtr;
+extern const u8 *gTeamToolboxBPtr;
+extern const u8 *gFieldItemMenuGroundTextPtr;
+extern const u8 *gUnknown_80FE940;
+extern const u8 *gWhichTextPtr1;
+
+#ifdef NONMATCHING
+
```
Comment (AnonymousRandomPerson) [inline src/code_805D8C8_1.c:159]: If you're ready to merge this in, add the decomp.me link as a comment here so others can easily pick it up.

**Comment by DizzyEggg** [comment]
Done

**Pull request merged**